pkg/cluster/config.go

```diff
@@ -32,6 +33,18 @@ const (
 	MachineAPINamespace = "openshift-machine-api"
 )
 
+var (
+	// acceptableProxyVars is a list of the supported proxy variables
+	acceptableProxyVars = []string{"HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY"}
+	// ClusterWideProxyVars is a map of the global egress proxy variables and values from WMCO's environment
+	ClusterWideProxyVars map[string]string
+)
+
+// init runs once, initializing global variables
+func init() {
+	ClusterWideProxyVars = getProxyVars()
```

Comment:
This is a one time initialization during WMCO bootup. What happens if these values are changed when WMCO is running?

Reply:
The WMCO is restarted by OLM when these values are updated. We never expected a user to update these values

Reply:
Please  add a comment.

Reply:
comment added explaining this